.changeset/clever-breads-share.md

```diff
@@ -0,0 +1,25 @@
+---
+"@fluidframework/core-interfaces": minor
+"fluid-framework": minor
+"@fluidframework/map": minor
+"@fluidframework/matrix": minor
+"@fluidframework/ordered-collection": minor
+"@fluidframework/register-collection": minor
+"@fluidframework/runtime-definitions": minor
+"@fluidframework/runtime-utils": minor
+"@fluidframework/sequence": minor
+"@fluid-experimental/sequence-deprecated": minor
+"@fluidframework/shared-object-base": minor
+"@fluidframework/synthesize": minor
+"@fluid-experimental/tree": minor
+"@fluidframework/tree": minor
+---
+
+Deprecated members of IFluidHandle are split off into new IFluidHandleInternal interface
+
+Split IFluidHandle into two interfaces, `IFluidHandle` and `IFluidHandleInternal`.
+Code depending on the previously deprecated members of IFluidHandle can access them by using `toFluidHandleInternal` from `@fluidframework/runtime-utils/legacy`.
```

Comment:
would FluidHandleLegacy be a better name than FluidHandleInternal?

Reply:
I don't think so. This type is intended to be use for the indefinite future since its what you have to implement to be a fluid handle and is used to implement our APIs related to handles. The only intended follow up change is that once we are confident existing external users have migrated off of the preciously deprecated members which are now here, this type will be moved from alpha to internal, but it will continue to exist as its needed by the implementation.

This documentation is for external users to help them update their codebase for this change if they were using the deprecated API, and is not for FF internal developers.